src/ServiceControl.AcceptanceTests/ConfigureEndpointAzureServiceBusStandardTopologyTransport.g.cs

```diff
@@ -0,0 +1,36 @@
+﻿extern alias TransportASBS;
```

Comment:
This comment is about the name of the file "ConfigureEndpointAzureServiceBusStandardTopologyTransport.g.cs"

This doesn't seem to be matching the name of the class.

Also, there is only one topology in ASBS, so I'm not sure it needs to be in the name.

Reply:
Also, why does this file have a ".g.cs" extension?

Reply:
g.cs or Generated.cs removes files from inspection thus preventing ReSharper inspection in failing the build

Reply:
I guess that works, but it seems like a weird way to prevent inspection errors.